index.js

```diff
@@ -22,3 +28,13 @@ export default fp(async (fastify, options) => {
 
   wrapResolvers(fastify.graphql.schema)
 })
+
+function isEnabled(options, { schema, source, context }) {
+  try {
```

Comment:
I'll add a check `options.enabled === false` before the other check.

Reply:
```suggestion
    if (!context.isEnabled) return
```
there is the check at the beginning of the file